### PR TITLE
test T2 T3 and T4

### PR DESCRIPTION
**Pull request by kuksak**
test T2 T3 and T4 added

### CODE REVIEW

File: tests/task_t_1.spec.js [hunk 1]
```diff
@@ -10,4 +10,41 @@ test('duck duck go T 1', async ({ page }) => {
...
+
+test('T3: Check that Cheat sheet for MSWord is displayed', async ({ page }) => {
+  await page.goto('https://duckduckgo.com/');
+  const searchInputField = page.locator('#search_form_input_homepage');
+  await searchInputField.fill('Microsoft word cheat sheet');
+  await page.click('#search_button_homepage');
+  const firstResult = await page.textContent('.c-base__title');
+  await expect(firstResult).toContain('Microsoft Word 2010');
+  await page.isVisible('span.chomp--link__mr');
+  expect(await page.click('span.chomp--link__mr')).click;
```
Comment (jevgenijusmarinuskinas) [inline tests/task_t_1.spec.js]: I see two clicks in this line, last click does not seem to be needed

File: tests/task_t_1.spec.js [hunk 2]
```diff
@@ -10,4 +10,41 @@ test('duck duck go T 1', async ({ page }) => {
...
+
+  expect(await page.isVisible('h6.cheatsheet__title:has-text("Formatting")'))
```
Comment (jevgenijusmarinuskinas) [inline tests/task_t_1.spec.js:35]: This should work fine, but I would suggest validating text separately, not inside of a selector for better code readability ant maintainability, e.g. `const textContent = await page.textContent('h6.cheatsheet__title')`

**Review by jevgenijusmarinuskinas — APPROVED** [review]

File: tests/task_t_1.spec.js [hunk 3]
```diff
@@ -10,4 +10,41 @@ test('duck duck go T 1', async ({ page }) => {
...
+  await searchInputField.fill('shorten www.wikipedia.com');
+  await page.click('#search_button_homepage');
+
+  const shortenURL = await page.inputValue('#shorten-url');
+  await page.goto(shortenURL);
+  await page.locator('#www-wikipedia-org').toBeVisible;
+  
+  const newPageURL = await page.url();
+  expect(newPageURL).toBe("https://www.wikipedia.org/");
 });
```
Comment (jevgenijusmarinuskinas) [inline tests/task_t_1.spec.js]: Missing new line

**Review by jevgenijusmarinuskinas — APPROVED** [review]

File: tests/task_t_1.spec.js (hunk 1 above)
Comment (kuksak) [reply tests/task_t_1.spec.js]: fixed

**Pull request merged**